src/main/java/uk/ac/ebi/subs/ena/processor/ENAProcessor.java

```diff
@@ -169,5 +190,89 @@ private String centerName(SubmissionEnvelope submissionEnvelope) {
         return paramMap;
     }
 
+    private Study setDefaultStudyReleaseDate(SubmissionEnvelope submissionEnvelope, Study study) {
+        Date subDate = submissionEnvelope.getSubmission().getSubmissionDate();
+
+        if (subDate != null && !study.isAccessioned() && study.getReleaseDate() == null) {
```

Comment:
this case is special because an accessioned study or sample means they have already been created and are being requested for an update. so here, we dont use submission date as its quite possible that the user who is requesting an update does not want to change release date. if they do want want to update it then they should specify the release date in the submittable object instead.

Reply:
OK, got it